server/lib/server/console-links.js

```diff
@@ -0,0 +1,65 @@
+/* Copyright (c) 2020 Red Hat, Inc. */
+const request = require('request').defaults({ rejectUnauthorized: false }),
+      config = require('../../config'),
+      fs = require('fs'),
+      log4js = require('log4js'),
+      logger = log4js.getLogger('service-account'),
+      serviceAccountPath = '/var/run/secrets/kubernetes.io/serviceaccount',
+      _ = require('lodash')
+
+module.exports.getConsoleLinks = (req, cb) => {
```

Comment:
What's the `req` here? I never see it used, but I'm not sure if I'm just missing something.